modules/templates/helper.go

```diff
@@ -623,14 +623,20 @@ func Str2html(raw string) template.HTML {
 }
 
 // Markdown2html render Markdown text to sanitized HTML
-func Markdown2html(raw string) template.HTML {
+func RenderMarkup(Type string, raw string) template.HTML {
 	var err error
 	var renderedContent string
-	if renderedContent, err = markdown.RenderString(&markup.RenderContext{}, raw); err != nil {
-		log.Warn("Markdown2html: Invalid markdown? %v", err)
-		return template.HTML(html.EscapeString(raw))
+
+	if Type == "markdown" {
+		if renderedContent, err = markdown.RenderString(&markup.RenderContext{}, raw); err != nil {
+			log.Warn("Markdown2html: Invalid markdown? %v", err)
+			return template.HTML(html.EscapeString(raw))
+		}
+	} else if Type == "markup" {
+		return template.HTML(markup.Sanitize(renderedContent))
```

Comment:
renderedContent here is always ""

Reply:
Yup, my bad, it was meant to be `raw` as in: if invalid markdown then at least show something (behavior as before this PR)